### PR TITLE
Update app.js a1 to implement 1

### LINKED ISSUE #3: There should be a goldfish
**Is your feature request related to a problem? Please describe.**
I'm always frustrated when there isn't a goldfish in this far to big aquarium.

**Describe the solution you'd like**
Buy a goldfish.

**Additional context**
Ref: https://aquariumtidings.com/wp-content/uploads/2014/06/Depositphotos_5125679_original.jpg

### PR DESCRIPTION
**Pull request by petute**
**Checklist:**

 
 - [x] Have you added an explanation of what your changes do and why you'd like them to be included?
 - [ ] Have you updated or added documentation for the change?
 - [ ] Have you tested your changes with successful results?

**Type of Changes:**

 
 - [x] Bug fix (non-breaking change which fixes an issue)
 - [ ] New feature (non-breaking change which adds functionality)
 - [ ] Breaking change (fix or feature that would cause existing functionality to change)

**What is the current behavior? (link to any open issues here)**
- Currently there is no goldfish. #3

**What is the new behavior (if this is a feature change)?**
- Now there is a goldfish. Closes #3  

**Other information**:
- Ref: https://www.asciiart.eu/animals/fish

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -31,6 +31,11 @@ class App extends React.Component {
 
 export default App;
 
+  ;,//;,    ,;/
+ o:::::::;;///
+>::::::::;;\\\
+  ''\\\\\'" ';\
```
Comment (kleberbaum) [inline src/App.js:39]: Please add a comment to explain the functionalities of this additions.

**Review by kleberbaum — CHANGES_REQUESTED** [review]
Should be ok to merge after a few changes.

**Review by petschna — APPROVED** [review]
Should be ok to merge.

**Review by kleberbaum — APPROVED** [review]
Should be ok to merge.

**Review by petute — COMMENTED** [review]
Should be ok to merge.

**Pull request merged**